meson.build

```
@@ -0,0 +1,53 @@
+project(
+  'hidapi',
+  'c',
+  version : '0.12.0',
```

```suggestion
  version : files('VERSION'),
```

The other build systems read the version from here, might as well do it for Meson too. ;)